src/PolyhedralGeometry/helpers.jl

```diff
@@ -30,6 +30,9 @@ IncidenceMatrix
 nrows(i::IncidenceMatrix) = Polymake.nrows(i)
 ncols(i::IncidenceMatrix) = Polymake.ncols(i)
 
+nrows(A::Polymake.Matrix) = Polymake.nrows(A)
```

Comment:
This actually has been added recently in https://github.com/oscar-system/Oscar.jl/pull/2530/commits/e9476c45ca503e3781e7cd92e7a56092ec775bf4#diff-b0d37454c5054ec64e2ac960fd319e08f74235cecbc97120ddf2afd94ba51453
Could you make sure to not include two different implementations?